ateam-common-packets/rust-lib/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+pub mod stspin_packets {
+	include!("stspin_bindings.rs");
```

Comment:
Any reason to use `include!` macro? You could just do `pub mod stspin_bindings;` (or change the generated file name to `stspin_packets.rs`)

Either way looks good to me and approving

Reply:
Tbh not really. I have no idea why the baseline documented example made some of the choices it did... I still much prefer the idea of running multiple bindgen processes if we ever get more than one header file in a "module" in which case the include approach would aggregate them and serve as a baseline lib.rs file that isn't autogenerated. Idk I think I'm okay with it but we can revisit later if we think it's becoming bad